app/models/miq_policy.rb

```diff
@@ -11,12 +11,26 @@ class MiqPolicy < ApplicationRecord
                                  PhysicalServer
                                  Vm).freeze
 
-  POLICY_TOWHAT_APPLIES_TO_CLASSES = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }.freeze
-
-  POLICY_MODES = {
-    "control" => _("Control"),
-    "compliance" => _("Compliance")
-  }.freeze
+  def self.POLICY_TOWHAT_APPLIES_TO_CLASSES(opts)
+    policyToWhatAppliesToClasses = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }
+    if opts.nil?
+      return policyToWhatAppliesToClasses
+    else
+      return policyToWhatAppliesToClasses[opts]
+    end
+  end.freeze
+
+  def self.POLICY_MODES(opts)
+    policyModes = {
+      "control" => N_("Control"),
+      "compliance" => N_("Compliance")
+    }
```

Comment:
These become normal `_()` calls

```suggestion
    policyModes = {
      "control" => _("Control"),
      "compliance" => _("Compliance")
    }
```